### ISSUE TITLE
Why do you crop images to (400X400) before training? what can go wrong if i train the network on orignal image of size (640x480)??)

### ISSUE THREAD

**Issue by ravan786** [NVlabs/Deep_Object_Pose#117]
Hi,
I was just wondering why do you randomly crop (400x400) patch before training? is it to make your model more robust to cases where entire object isn’t present in the image? what can go wrong if i train the model on original image?

**Comment by TontonTremblay** [comment]
Yes that is a good point for making more robust. You can see this as a form of data augmentation. The original images are quite large, You can for sure train on it, I would make sure the targets are generated correctly (I think I had hard coded some values in this version of the code). Also make sure you can fit the whole thing during training. Moreover inference will be slower.
